src/resources/companyResources.ts

```diff
@@ -0,0 +1,254 @@
+import { ResourceIF } from '@/interfaces'
```

Comment:
How about having this file importing sub-files for all different entity types?

Reply:
Yes, I'd like to see a simpler declaration here (ie, hide details in the imports).

Reply:
For sure, will break up each entity!

Reply:
Updated!